docs/examples_literate/mixed_integer/aux_files/antidiag.jl

```diff
@@ -79,31 +79,27 @@ antidiag(x::AbstractExpr, k::Int = 0) = AntidiagAtom(x, k)
 # 3. We populate coeff with 1s at the correct indices
 # The canonical form will then be:
 # coeff * x - d = 0
-function conic_form!(
+function Convex._conic_form!(
```

Comment:
@ericphanson This cannot be an internal function if it's something that can be implemented by extensions

Reply:
What about `new_conic_form!` ?

Reply:
I have renamed it in https://github.com/jump-dev/Convex.jl/pull/504/commits/a8b44b212d822552221526f4cbd011f30336b73a